docs/docs/dev-manuals/services/media.mdx

```diff
@@ -0,0 +1,87 @@
+import CustomImage from "@site/src/components/CustomImage";
+
+# Media Service
+
+The media service provides the bounded context `Media`.
+It is responsible for managing medias.
```

Comment:
```suggestion
It is responsible for managing assets.
```

Reply:
Would it be ok to change it to "media files"? Assets could be confused with financial assets

Reply:
Sounds fine